src/spikeinterface/core/baserecordingsnippets.py

```diff
@@ -275,8 +280,34 @@ def set_dummy_probe_from_locations(self, locations, shape="circle", shape_params
         if ndim == 3:
             probe = probe.to_3d(axes=axes)
 
+        return probe
+
+    def set_dummy_probe_from_locations(self, locations, shape="circle", shape_params={"radius": 1}, axes="xy"):
+        """
+        Sets a 'dummy' probe based on locations.
+
+        Parameters
+        ----------
+        locations : np.array
+            Array with channel locations (num_channels, ndim) [ndim can be 2 or 3]
+        shape : str, optional
+            Electrode shapes, by default "circle"
+        shape_params : dict, optional
+            Shape parameters, by default {"radius": 1}
+        axes : str, optional
+            If ndim is 3, indicates the axes that define the plane of the electrodes, by default "xy"
+        """
+        probe = self.create_dummy_probe_from_locations(locations, shape=shape, shape_params=shape_params, axes=axes)
         self.set_probe(probe, in_place=True)
 
+    def set_dummy_probe(self):
```

Comment:
What is the use case of this ?
I think having a wrirng probe automatically is dangerous. I think the end have to do it himself.